cmd/ndm_daemonset/probe/udevprobe_test.go

```diff
@@ -186,8 +190,12 @@ func TestUdevProbe(t *testing.T) {
 	}
 	probeEvent.addBlockDeviceEvent(eventDetails)
 	// Retrieve disk resource
-	uuid, _ := generateUUID(*deviceDetails)
+	uuid, ok := generateUUID(*deviceDetails)
+	if !ok {
+		t.Skip("cannot uniquely identify disk. skipping")
```

Comment:
Along with skipping this test, can you add another test case, in which the uuid will be generated and the blockdevice resource will be created. If we skip this test, I think there is no test available for `addBlockDeviceEvent`

Reply:
Sure. I'll have to see what test I can add and how I can make it work.